docs/Development/Committer.md

```diff
@@ -0,0 +1,76 @@
+
+The [PMC](https://www.apache.org/foundation/how-it-works.html#pmc-members) makes someone a committer via nomination, discussion, and then majority vote. We use data from as many sources as possible to inform our reasoning. Here are some examples:
+
+- Git metrics for [incubator-iotdb](https://github.com/apache/incubator-iotdb/graphs/contributors) 
+- Code reviews given and received on [incubator-iotdb](https://github.com/apache/incubator-iotdb/pulls)
+- Clear areas of ownership (a runner, a DSL, IO connector, documentation, etc.)
+- Public events
+- Firsthand PMC testimonials
+
+The PMC has assembled the following set of guidelines for becoming a committer.
+
+## An Apache IoTDBcommitter…
+
+### Takes many forms
+
+There are many actions other than coding that build the trust we place in a committer - code review, design discussion, user support, community outreach, improving infrastructure, documentation, project management, etc.
+
+### Knows, upholds, and reinforces the Apache Software Foundation code of conduct
+
+See the [ASF documentation](https://www.apache.org/foundation/policies/conduct.html). In particular, they manifestly strive to:
+
+- Be open
+- Be empathetic
+- Be welcoming
+- Be friendly
+- Be patient
+- Be collaborative
+- Be inquisitive
+- Be careful in the words that they choose
+
+### Knows, upholds, and reinforces the responsibilities of an Apache Software Foundation committer
+
+See the [ASF documentation](https://www.apache.org/dev/committers.html#committer-responsibilities).
+
+- They help create a product that will outlive the interest of any particular volunteer (including themselves)
+- They grow and maintain the health of the Apache community
+- They help out with surrounding work, such as the website & documentation
+- They help users
+- They can be trusted to decide when code is ready for release, or when to ask someone else to make the judgment
+- They can be trusted to decide when to merge code (if a code contributor) or when to ask someone else to make the judgment
+
+### Knows, upholds, and reinforces the Beam community’s practices
+
+- They have a proven commitment to the project
+- They share their intentions with the community
+- They accept and integrate community feedback in their plans, designs, code, etc.
+- They earnestly try to make Beam better with their contributions
+- In particular, if a code contributor:
+  - They earnestly try to make IoTDB better with their own code
+  - They earnestly try to make IoTDB better with code review
+  - They accept and integrate feedback on their code
+  - They know, follow, and enforce IoTDB's practices while reviewing/merging code - style, documentation, testing, backward compatibility, etc.
```

Comment:
Finally, please add a paragraph:
The code of conduct is derived from [Apache Beam](https://beam.apache.org/contribute/become-a-committer/ ).